src/main/java/com/example/boxinator/Controllers/ShipmentController.java

```diff
@@ -42,12 +47,11 @@
         CommonResponse cr = new CommonResponse();
```

Comment:
getShipment ger "500 Ambiguous handler methods mapped for ...". Kan vara endpoint som är snarlik i anropet, ex  metoden getAllCanceledShipments eller getAllCompletedShipments men är inte säker vad det beror på